test/integration/patients/patient/archive.js

```diff
@@ -419,4 +419,101 @@ context('patient archive page', function() {
       .find('button')
       .should('not.exist');
   });
+
+  specify('work with only work:team:manage permission', function() {
```

Comment:
I don't think we need the "only"s in the description.  There's already contextual knowledge required as this user does have other permissions, so this means "only" one `work:__:manage` permission, and currently it is a requirement that all users only have one of that type of permission and it's doubtful we'd ever support a combination.  Although perhaps ironically, they would functionally work to have any combination.. But the more permissible will always win.

Reply:
> Although perhaps ironically, they would functionally work to have any combination.. **But the more permissible will always win.**